src/pjpersist/tests/test_serialize.py

```diff
@@ -421,6 +421,15 @@ def doctest_ObjectWriter_get_state_mappings():
 
       >>> pprint.pprint(writer.get_state({1: 'one', 2: 'two', 3: 'three'}))
       {'dict_data': [(1, 'one'), (2, 'two'), (3, 'three')]}
+
+    Another edge case is when the input has a `dict_data` key
+
+      >>> writer = serialize.ObjectWriter(None)
+      >>> state = writer.get_state(
+      ...     {'1': 1, 'dict_data': 'works?'})
+      >>> pprint.pprint(sorted(state['dict_data']))
+      [('1', 1), ('dict_data', 'works?')]
```

Comment:
Do we need to test an edge case where there is a `"list_data"` in the input?

Reply:
don't think so, we don't add `list_data`

Reply:
Why not?